lib/apipie/param_description.rb

```diff
@@ -120,11 +120,11 @@ def validate(value)
       return true if allow_nil && value.nil?
       return true if allow_blank && value.blank?
       value = normalized_value(value)
-      if (!allow_nil && value.nil?) || (blank_forbidden? && value.blank?) || !validator.valid?(value)
```

Comment:
I don't know if we can remove the `!allow_nil` check and the `blank_forbidden?` check.
I think they were added for a reason...

I don't understand what problem we are trying to fix here.